manifests/v1beta1/components/controller/rbac.yaml

```diff
@@ -7,19 +7,38 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - configmaps
-      - serviceaccounts
       - services
       - events
-      - namespaces
+    verbs:
+      - "*"
```

Comment:
Do we need `*` for services ?

Reply:
Yes. We do it in  https://github.com/kubeflow/katib/blob/master/pkg/controller.v1beta1/suggestion/suggestion_controller_util.go

Reply:
We do get/create/delete/watch